ROADMAP.md

```diff
@@ -0,0 +1,23 @@
+# Roadmap
+
+## Top-level goals
+
+* Extensibility: custom components, shaders
+* Performance: raycasting (three.js), head tracking on mobile (webvr-polyfill), anything DOM/Custom Elements-related
+* Ecosystem: enable discovery and distribution of assets, components, scenes, snippets, shaders
+
+## `0.2.0`
+
+* Merge aframe-core repo into aframe repo
+* Custom shaders
+* New primitives implementation (extending entities instead of templates of grouped elements)
+* Video playback issues (on Android, iOS)
+* Improved mobile experience (orientation lock, initial position, etc.)
+* Systems (shared logic of components)
+* Texture caching
+* Improved asset management and loading
+* obj model support
+
+## Future
+
+* glTF support
```

Comment:
we can get the initial glTF loader in. there are glTF converters. but, can probably hold off until ecosystem improves.

but look at dat perf:

![image](https://cloud.githubusercontent.com/assets/203725/13091812/c93df540-d4b1-11e5-801c-80bb107f4626.png)